auth_dynamic_groups/model/res_users.py

```diff
@@ -35,10 +35,9 @@ def _login(self, db, login, password):
         return uid
 
     def update_dynamic_groups(self, uid, db):
-        pool = RegistryManager.get(db)
-        cr = pool._db.cursor()
-        user = pool.get('res.users').browse(cr, SUPERUSER_ID, uid)
-        groups_obj = pool.get('res.groups')
+        cr = self.pool._db.cursor(serialized=True)
```

Comment:
@moylop260 thanks so much, I messed up the only thing that was to mess up here...